code/modules/surgery/organs_internal.dm

```diff
@@ -297,10 +324,12 @@
 
 		// Extract the organ!
 		if(target.op_stage.current_organ)
-			var/obj/item/organ/O = target.op_stage.organ_ref
+			var/obj/item/organ/internal/O = target.op_stage.organ_ref
 			if(O && istype(O))
+
 				spread_germs_to_organ(O, user)
-				O.removed(user)
+				O.Remove(user)
```

Comment:
surgeon: "And now, vee carefully remove zeh brain from zeh surgeon's head"

*the surgeon seizes up and falls limp, his eyes dead and lifeless*